src/Tribe/Events/Attendees_List.php

```diff
@@ -153,7 +186,74 @@ public function maybe_update_attendee_list_hide_meta( $post ) {
 			$post
 		);
 
-		return update_post_meta( $post->ID, Tribe__Tickets_Plus__Attendees_List::HIDE_META_KEY, (int) $is_showing_attendee_list );
+		return update_post_meta( $post->ID, self::HIDE_META_KEY, (int) $is_showing_attendee_list );
+	}
+
+	/**
+	 * Get list of public attendees for display.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post|int $post_id Post object or ID.
+	 * @param  int        $limit   Limit of attendees to be retrieved from database.
+	 *
+	 * @return array List of public attendees for display.
+	 */
+	public function get_attendees_for_post( $post_id, $limit = - 1 ) {
+		$post = get_post( $post_id );
+
+		if ( ! $post instanceof WP_Post ) {
+			return [];
+		}
+
+		$args = [
+			'by' => [
+				// Exclude people who have opted out or not specified optout.
+				'optout' => 'no_or_none',
+				// Only include public attendees.
+				'post_status' => 'publish',
+				// Only include RSVP status yes.
+				'rsvp_status__or_none' => 'yes',
+				// Only include public order statuses.
+				'order_status' => 'public',
+			],
+		];
+
+		/**
+		 * Allow for adjusting the limit of attendees fetched from the database for the front-end "Who's Coming?" list.
+		 *
+		 * @since 4.10.6
+		 *
+		 * @param int $limit_attendees Number of attendees to retrieve. Default is no limit -1.
+		 */
+		$limit_attendees = (int) apply_filters( 'tribe_tickets_attendees_list_limit_attendees', $limit );
+
+		if ( 0 < $limit_attendees ) {
+			$args['per_page'] = $limit_attendees;
+		}
+
+		$attendees  = \Tribe__Tickets__Tickets::get_event_attendees( $post->ID, $args );
+		$emails     = [];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 5 spaces

<sub>hash: e8fea7f7874e3bc0551e188939e97a0b</sub>